src/basic.h

```diff
@@ -270,4 +270,3 @@ const static TypeID type_code_id = ID; \
 virtual TypeID get_type_code() const { return type_code_id; };
 
 #endif
-
```

Comment:
@mafiya69 You accidentally deleted this line. Can you please add the new line again.